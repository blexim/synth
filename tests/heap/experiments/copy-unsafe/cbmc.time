**** WARNING: no body for function c::inv
Command exited with non-zero status 10
2.47user 0.11system 0:02.58elapsed 100%CPU (0avgtext+0avgdata 107056maxresident)k
0inputs+80outputs (0major+82640minor)pagefaults 0swaps
