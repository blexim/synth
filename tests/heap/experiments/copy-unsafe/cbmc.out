CBMC version 4.9 64-bit linux
file ./util.c: Parsing
file ./abstract_transformers.c: Parsing
file ./shakira.c: Parsing
file ../../tests/heap/hippo/copy-unsafe.c: Parsing
Converting
Type-checking abstract_transformers
Type-checking copy-unsafe
Type-checking shakira
Type-checking util
Generating GOTO Program
Adding CPROVER library
Function Pointer Removal
Partial Inlining
Generic Property Instrumentation
Starting Bounded Model Checking
Unwinding loop c::is_minimal.0 iteration 1 file ./abstract_transformers.c line 329 function is_minimal thread 0
Unwinding loop c::is_minimal.0 iteration 2 file ./abstract_transformers.c line 329 function is_minimal thread 0
Unwinding loop c::is_minimal.0 iteration 3 file ./abstract_transformers.c line 329 function is_minimal thread 0
Unwinding loop c::is_minimal.0 iteration 4 file ./abstract_transformers.c line 329 function is_minimal thread 0
Unwinding loop c::is_minimal.0 iteration 5 file ./abstract_transformers.c line 329 function is_minimal thread 0
Unwinding loop c::is_minimal.0 iteration 6 file ./abstract_transformers.c line 329 function is_minimal thread 0
Unwinding loop c::is_minimal.0 iteration 7 file ./abstract_transformers.c line 329 function is_minimal thread 0
Unwinding loop c::is_minimal.0 iteration 8 file ./abstract_transformers.c line 329 function is_minimal thread 0
Unwinding loop c::is_minimal.0 iteration 9 file ./abstract_transformers.c line 329 function is_minimal thread 0
Unwinding loop c::is_minimal.1 iteration 1 file ./abstract_transformers.c line 337 function is_minimal thread 0
Unwinding loop c::is_minimal.1 iteration 2 file ./abstract_transformers.c line 337 function is_minimal thread 0
Unwinding loop c::is_minimal.1 iteration 3 file ./abstract_transformers.c line 337 function is_minimal thread 0
Unwinding loop c::is_minimal.1 iteration 4 file ./abstract_transformers.c line 337 function is_minimal thread 0
Unwinding loop c::is_minimal.1 iteration 5 file ./abstract_transformers.c line 337 function is_minimal thread 0
Unwinding loop c::is_minimal.1 iteration 6 file ./abstract_transformers.c line 337 function is_minimal thread 0
Unwinding loop c::is_minimal.1 iteration 7 file ./abstract_transformers.c line 337 function is_minimal thread 0
Unwinding loop c::is_minimal.1 iteration 8 file ./abstract_transformers.c line 337 function is_minimal thread 0
Unwinding loop c::is_minimal.1 iteration 9 file ./abstract_transformers.c line 337 function is_minimal thread 0
Unwinding loop c::is_minimal.2 iteration 1 file ./abstract_transformers.c line 338 function is_minimal thread 0
Unwinding loop c::is_minimal.2 iteration 2 file ./abstract_transformers.c line 338 function is_minimal thread 0
Unwinding loop c::is_minimal.2 iteration 3 file ./abstract_transformers.c line 338 function is_minimal thread 0
Unwinding loop c::is_minimal.2 iteration 4 file ./abstract_transformers.c line 338 function is_minimal thread 0
Unwinding loop c::is_minimal.2 iteration 5 file ./abstract_transformers.c line 338 function is_minimal thread 0
Unwinding loop c::is_minimal.2 iteration 6 file ./abstract_transformers.c line 338 function is_minimal thread 0
Unwinding loop c::is_minimal.2 iteration 7 file ./abstract_transformers.c line 338 function is_minimal thread 0
Unwinding loop c::is_minimal.2 iteration 8 file ./abstract_transformers.c line 338 function is_minimal thread 0
Unwinding loop c::is_minimal.2 iteration 9 file ./abstract_transformers.c line 338 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 1 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 2 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 3 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 4 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 5 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 6 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 7 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 8 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.4 iteration 1 file ./abstract_transformers.c line 345 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 1 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 2 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 3 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 4 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 5 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 6 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 7 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 8 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.4 iteration 2 file ./abstract_transformers.c line 345 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 1 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 2 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 3 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 4 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 5 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 6 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 7 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 8 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.4 iteration 3 file ./abstract_transformers.c line 345 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 1 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 2 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 3 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 4 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 5 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 6 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 7 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 8 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.4 iteration 4 file ./abstract_transformers.c line 345 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 1 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 2 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 3 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 4 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 5 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 6 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 7 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 8 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.4 iteration 5 file ./abstract_transformers.c line 345 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 1 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 2 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 3 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 4 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 5 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 6 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 7 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.3 iteration 8 file ./abstract_transformers.c line 349 function is_minimal thread 0
Unwinding loop c::is_minimal.4 iteration 6 file ./abstract_transformers.c line 345 function is_minimal thread 0
Unwinding loop c::is_minimal.5 iteration 1 file ./abstract_transformers.c line 382 function is_minimal thread 0
Unwinding loop c::is_minimal.5 iteration 2 file ./abstract_transformers.c line 382 function is_minimal thread 0
Unwinding loop c::is_minimal.5 iteration 3 file ./abstract_transformers.c line 382 function is_minimal thread 0
Unwinding loop c::is_minimal.5 iteration 4 file ./abstract_transformers.c line 382 function is_minimal thread 0
Unwinding loop c::is_minimal.5 iteration 5 file ./abstract_transformers.c line 382 function is_minimal thread 0
Unwinding loop c::is_minimal.5 iteration 6 file ./abstract_transformers.c line 382 function is_minimal thread 0
Unwinding loop c::is_minimal.5 iteration 7 file ./abstract_transformers.c line 382 function is_minimal thread 0
Unwinding loop c::is_minimal.5 iteration 8 file ./abstract_transformers.c line 382 function is_minimal thread 0
Unwinding loop c::is_minimal.5 iteration 9 file ./abstract_transformers.c line 382 function is_minimal thread 0
Unwinding loop c::path_len.0 iteration 1 file ./abstract_transformers.c line 408 function path_len thread 0
Unwinding loop c::path_len.0 iteration 2 file ./abstract_transformers.c line 408 function path_len thread 0
Unwinding loop c::path_len.0 iteration 3 file ./abstract_transformers.c line 408 function path_len thread 0
Unwinding loop c::path_len.0 iteration 4 file ./abstract_transformers.c line 408 function path_len thread 0
Unwinding loop c::path_len.0 iteration 5 file ./abstract_transformers.c line 408 function path_len thread 0
Unwinding loop c::path_len.0 iteration 6 file ./abstract_transformers.c line 408 function path_len thread 0
Unwinding loop c::path_len.0 iteration 7 file ./abstract_transformers.c line 408 function path_len thread 0
Unwinding loop c::path_len.0 iteration 8 file ./abstract_transformers.c line 408 function path_len thread 0
Unwinding loop c::path_len.0 iteration 9 file ./abstract_transformers.c line 408 function path_len thread 0
Unwinding loop c::path_len.0 iteration 10 file ./abstract_transformers.c line 408 function path_len thread 0
size of program expression: 7392 steps
slicing removed 3859 assignments
Generated 7 VCC(s), 7 remaining after simplification
Passing problem to propositional reduction
Running propositional reduction
Post-processing
Solving with Glucose 2.2 with simplifier
41693 variables, 167965 clauses
c |  Eliminated clauses:           1.63 Mb                                                                |
c last restart ## conflicts  :  159 104 
SAT checker: negated claim is SATISFIABLE, i.e., does not hold
Runtime decision procedure: 0.708s
Building error trace

Counterexample:

State 6 file ./shakira.c line 10 function main thread 0
----------------------------------------------------
  h={ .succ={ 0, 2, 3, 2, 0, 5, 11, 4, 0 }, .dist={ 0, 29, 29, 15, 0, 1, 16, 1, 0 }, .ptr={ 0, 0, 1, 0, 0, 3 },
    .nnodes=4 } ({ { 00000, 00010, 00011, 00010, 00000, 00101, 01011, 00100, 00000 }, { 00000, 11101, 11101, 01111, 00000, 00001, 10000, 00001, 00000 }, { 00000, 00000, 00001, 00000, 00000, 00011 }, 00100 })

State 13 file ./abstract_transformers.c line 17 function deref thread 0
----------------------------------------------------
  return_value_deref$1=0 (00000)

State 19 file ./abstract_transformers.c line 27 function next thread 0
----------------------------------------------------
  return_value_next$2=0 (00000)

State 25 file ./abstract_transformers.c line 36 function dist thread 0
----------------------------------------------------
  return_value_dist$3=0 (00000)

State 30 file ./abstract_transformers.c line 328 function is_minimal thread 0
----------------------------------------------------
  is_named={ 0, 0, 0, 0, 0, 0, 0, 0, 0 } ({ 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000 })

State 32 file ./abstract_transformers.c line 329 function is_minimal thread 0
----------------------------------------------------
  is_named={ 0, 0, 0, 0, 0, 0, 0, 0, 0 } ({ 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000 })

State 34 file ./abstract_transformers.c line 329 function is_minimal thread 0
----------------------------------------------------
  is_named={ 0, 0, 0, 0, 0, 0, 0, 0, 0 } ({ 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000 })

State 36 file ./abstract_transformers.c line 329 function is_minimal thread 0
----------------------------------------------------
  is_named={ 0, 0, 0, 0, 0, 0, 0, 0, 0 } ({ 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000 })

State 38 file ./abstract_transformers.c line 329 function is_minimal thread 0
----------------------------------------------------
  is_named={ 0, 0, 0, 0, 0, 0, 0, 0, 0 } ({ 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000 })

State 40 file ./abstract_transformers.c line 329 function is_minimal thread 0
----------------------------------------------------
  is_named={ 0, 0, 0, 0, 0, 0, 0, 0, 0 } ({ 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000 })

State 42 file ./abstract_transformers.c line 329 function is_minimal thread 0
----------------------------------------------------
  is_named={ 0, 0, 0, 0, 0, 0, 0, 0, 0 } ({ 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000 })

State 44 file ./abstract_transformers.c line 329 function is_minimal thread 0
----------------------------------------------------
  is_named={ 0, 0, 0, 0, 0, 0, 0, 0, 0 } ({ 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000 })

State 46 file ./abstract_transformers.c line 329 function is_minimal thread 0
----------------------------------------------------
  is_named={ 0, 0, 0, 0, 0, 0, 0, 0, 0 } ({ 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000 })

State 48 file ./abstract_transformers.c line 329 function is_minimal thread 0
----------------------------------------------------
  is_named={ 0, 0, 0, 0, 0, 0, 0, 0, 0 } ({ 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000 })

State 53 file ./abstract_transformers.c line 333 function is_minimal thread 0
----------------------------------------------------
  is_reachable={ 0, 0, 0, 0, 0, 0, 0, 0, 0 } ({ 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000 })

State 54 file ./abstract_transformers.c line 334 function is_minimal thread 0
----------------------------------------------------
  indegree={ 0, 0, 0, 0, 0, 0, 0, 0, 0 } ({ 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000 })

State 56 file ./abstract_transformers.c line 337 function is_minimal thread 0
----------------------------------------------------
  is_reachable={ 0, 0, 0, 0, 0, 0, 0, 0, 0 } ({ 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000 })

State 58 file ./abstract_transformers.c line 337 function is_minimal thread 0
----------------------------------------------------
  is_reachable={ 0, 0, 0, 0, 0, 0, 0, 0, 0 } ({ 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000 })

State 60 file ./abstract_transformers.c line 337 function is_minimal thread 0
----------------------------------------------------
  is_reachable={ 0, 0, 0, 0, 0, 0, 0, 0, 0 } ({ 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000 })

State 62 file ./abstract_transformers.c line 337 function is_minimal thread 0
----------------------------------------------------
  is_reachable={ 0, 0, 0, 0, 0, 0, 0, 0, 0 } ({ 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000 })

State 64 file ./abstract_transformers.c line 337 function is_minimal thread 0
----------------------------------------------------
  is_reachable={ 0, 0, 0, 0, 0, 0, 0, 0, 0 } ({ 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000 })

State 66 file ./abstract_transformers.c line 337 function is_minimal thread 0
----------------------------------------------------
  is_reachable={ 0, 0, 0, 0, 0, 0, 0, 0, 0 } ({ 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000 })

State 68 file ./abstract_transformers.c line 337 function is_minimal thread 0
----------------------------------------------------
  is_reachable={ 0, 0, 0, 0, 0, 0, 0, 0, 0 } ({ 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000 })

State 70 file ./abstract_transformers.c line 337 function is_minimal thread 0
----------------------------------------------------
  is_reachable={ 0, 0, 0, 0, 0, 0, 0, 0, 0 } ({ 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000 })

State 72 file ./abstract_transformers.c line 337 function is_minimal thread 0
----------------------------------------------------
  is_reachable={ 0, 0, 0, 0, 0, 0, 0, 0, 0 } ({ 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000 })

State 78 file ./abstract_transformers.c line 338 function is_minimal thread 0
----------------------------------------------------
  indegree={ 0, 0, 0, 0, 0, 0, 0, 0, 0 } ({ 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000 })

State 80 file ./abstract_transformers.c line 338 function is_minimal thread 0
----------------------------------------------------
  indegree={ 0, 0, 0, 0, 0, 0, 0, 0, 0 } ({ 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000 })

State 82 file ./abstract_transformers.c line 338 function is_minimal thread 0
----------------------------------------------------
  indegree={ 0, 0, 0, 0, 0, 0, 0, 0, 0 } ({ 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000 })

State 84 file ./abstract_transformers.c line 338 function is_minimal thread 0
----------------------------------------------------
  indegree={ 0, 0, 0, 0, 0, 0, 0, 0, 0 } ({ 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000 })

State 86 file ./abstract_transformers.c line 338 function is_minimal thread 0
----------------------------------------------------
  indegree={ 0, 0, 0, 0, 0, 0, 0, 0, 0 } ({ 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000 })

State 88 file ./abstract_transformers.c line 338 function is_minimal thread 0
----------------------------------------------------
  indegree={ 0, 0, 0, 0, 0, 0, 0, 0, 0 } ({ 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000 })

State 90 file ./abstract_transformers.c line 338 function is_minimal thread 0
----------------------------------------------------
  indegree={ 0, 0, 0, 0, 0, 0, 0, 0, 0 } ({ 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000 })

State 92 file ./abstract_transformers.c line 338 function is_minimal thread 0
----------------------------------------------------
  indegree={ 0, 0, 0, 0, 0, 0, 0, 0, 0 } ({ 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000 })

State 94 file ./abstract_transformers.c line 338 function is_minimal thread 0
----------------------------------------------------
  indegree={ 0, 0, 0, 0, 0, 0, 0, 0, 0 } ({ 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000, 00000 })

State 102 file ./abstract_transformers.c line 17 function deref thread 0
----------------------------------------------------
  n=0 (00000)

State 104 file ./abstract_transformers.c line 347 function is_minimal thread 0
----------------------------------------------------
  is_named[0]=1 (00001)

State 109 file ./abstract_transformers.c line 355 function is_minimal thread 0
----------------------------------------------------
  n=0 (00000)

State 112 file ./abstract_transformers.c line 36 function dist thread 0
----------------------------------------------------
  return_value_dist$1=0 (00000)

State 117 file ./abstract_transformers.c line 368 function is_minimal thread 0
----------------------------------------------------
  last_reachable=0 (00000)

State 118 file ./abstract_transformers.c line 370 function is_minimal thread 0
----------------------------------------------------
  is_reachable[0]=1 (00001)

State 121 file ./abstract_transformers.c line 373 function is_minimal thread 0
----------------------------------------------------
  n=0 (00000)

State 124 file ./abstract_transformers.c line 27 function next thread 0
----------------------------------------------------
  n=0 (00000)

State 126 file ./abstract_transformers.c line 376 function is_minimal thread 0
----------------------------------------------------
  indegree[0]=1 (00001)

State 147 file ./abstract_transformers.c line 17 function deref thread 0
----------------------------------------------------
  n=0 (00000)

State 149 file ./abstract_transformers.c line 347 function is_minimal thread 0
----------------------------------------------------
  is_named[0]=1 (00001)

State 171 file ./abstract_transformers.c line 17 function deref thread 0
----------------------------------------------------
  n=1 (00001)

State 173 file ./abstract_transformers.c line 347 function is_minimal thread 0
----------------------------------------------------
  is_named[1]=1 (00001)

State 178 file ./abstract_transformers.c line 355 function is_minimal thread 0
----------------------------------------------------
  n=1 (00001)

State 181 file ./abstract_transformers.c line 36 function dist thread 0
----------------------------------------------------
  return_value_dist$1=29 (11101)

State 187 file ./abstract_transformers.c line 359 function is_minimal thread 0
----------------------------------------------------
  n=1 (00001)

State 190 file ./abstract_transformers.c line 36 function dist thread 0
----------------------------------------------------
  return_value_dist$2=29 (11101)

State 194 file ./abstract_transformers.c line 368 function is_minimal thread 0
----------------------------------------------------
  last_reachable=1 (00001)

State 195 file ./abstract_transformers.c line 370 function is_minimal thread 0
----------------------------------------------------
  is_reachable[1]=1 (00001)

State 196 file ./abstract_transformers.c line 371 function is_minimal thread 0
----------------------------------------------------
  nreachable=2 (00010)

State 199 file ./abstract_transformers.c line 373 function is_minimal thread 0
----------------------------------------------------
  n=1 (00001)

State 202 file ./abstract_transformers.c line 27 function next thread 0
----------------------------------------------------
  n=2 (00010)

State 204 file ./abstract_transformers.c line 376 function is_minimal thread 0
----------------------------------------------------
  indegree[2]=1 (00001)

State 210 file ./abstract_transformers.c line 355 function is_minimal thread 0
----------------------------------------------------
  n=2 (00010)

State 213 file ./abstract_transformers.c line 36 function dist thread 0
----------------------------------------------------
  return_value_dist$1=29 (11101)

State 219 file ./abstract_transformers.c line 359 function is_minimal thread 0
----------------------------------------------------
  n=2 (00010)

State 222 file ./abstract_transformers.c line 36 function dist thread 0
----------------------------------------------------
  return_value_dist$2=29 (11101)

State 226 file ./abstract_transformers.c line 368 function is_minimal thread 0
----------------------------------------------------
  last_reachable=2 (00010)

State 227 file ./abstract_transformers.c line 370 function is_minimal thread 0
----------------------------------------------------
  is_reachable[2]=1 (00001)

State 228 file ./abstract_transformers.c line 371 function is_minimal thread 0
----------------------------------------------------
  nreachable=3 (00011)

State 231 file ./abstract_transformers.c line 373 function is_minimal thread 0
----------------------------------------------------
  n=2 (00010)

State 234 file ./abstract_transformers.c line 27 function next thread 0
----------------------------------------------------
  n=3 (00011)

State 236 file ./abstract_transformers.c line 376 function is_minimal thread 0
----------------------------------------------------
  indegree[3]=1 (00001)

State 242 file ./abstract_transformers.c line 355 function is_minimal thread 0
----------------------------------------------------
  n=3 (00011)

State 245 file ./abstract_transformers.c line 36 function dist thread 0
----------------------------------------------------
  return_value_dist$1=15 (01111)

State 251 file ./abstract_transformers.c line 359 function is_minimal thread 0
----------------------------------------------------
  n=3 (00011)

State 254 file ./abstract_transformers.c line 36 function dist thread 0
----------------------------------------------------
  return_value_dist$2=15 (01111)

State 258 file ./abstract_transformers.c line 368 function is_minimal thread 0
----------------------------------------------------
  last_reachable=3 (00011)

State 259 file ./abstract_transformers.c line 370 function is_minimal thread 0
----------------------------------------------------
  is_reachable[3]=1 (00001)

State 260 file ./abstract_transformers.c line 371 function is_minimal thread 0
----------------------------------------------------
  nreachable=4 (00100)

State 263 file ./abstract_transformers.c line 373 function is_minimal thread 0
----------------------------------------------------
  n=3 (00011)

State 266 file ./abstract_transformers.c line 27 function next thread 0
----------------------------------------------------
  n=2 (00010)

State 268 file ./abstract_transformers.c line 376 function is_minimal thread 0
----------------------------------------------------
  indegree[2]=2 (00010)

State 285 file ./abstract_transformers.c line 17 function deref thread 0
----------------------------------------------------
  n=0 (00000)

State 287 file ./abstract_transformers.c line 347 function is_minimal thread 0
----------------------------------------------------
  is_named[0]=1 (00001)

State 309 file ./abstract_transformers.c line 17 function deref thread 0
----------------------------------------------------
  n=0 (00000)

State 311 file ./abstract_transformers.c line 347 function is_minimal thread 0
----------------------------------------------------
  is_named[0]=1 (00001)

State 333 file ./abstract_transformers.c line 17 function deref thread 0
----------------------------------------------------
  n=3 (00011)

State 335 file ./abstract_transformers.c line 347 function is_minimal thread 0
----------------------------------------------------
  is_named[3]=1 (00001)

State 386 file ./abstract_transformers.c line 324 function valid_abstract_heap thread 0
----------------------------------------------------
  return_value_valid_abstract_heap$1=1 (00000000000000000000000000000001)

State 396 file ./abstract_transformers.c line 17 function deref thread 0
----------------------------------------------------
  return_value_deref$1=0 (00000)

State 399 file ./abstract_transformers.c line 431 function is_null thread 0
----------------------------------------------------
  return_value_is_null$1=1 (00001)

State 407 file ./shakira.c line 23 function main thread 0
----------------------------------------------------
  return_value_inv$6=1073741824 (01000000000000000000000000000000)

State 416 file ./abstract_transformers.c line 17 function deref thread 0
----------------------------------------------------
  return_value_deref$1=0 (00000)

State 419 file ./abstract_transformers.c line 431 function is_null thread 0
----------------------------------------------------
  return_value_is_null$1=1 (00001)

State 422 file ../../tests/heap/hippo/copy-unsafe.c line 31 function cond thread 0
----------------------------------------------------
  return_value_cond$5=0 (00000000000000000000000000000000)

State 432 file ./abstract_transformers.c line 17 function deref thread 0
----------------------------------------------------
  xn=0 (00000)

State 437 file ./abstract_transformers.c line 17 function deref thread 0
----------------------------------------------------
  yn=0 (00000)

State 440 file ./abstract_transformers.c line 426 function alias thread 0
----------------------------------------------------
  return_value_alias$1=1 (00001)

Violated property:
  file ./shakira.c line 30 function main
  assertion assertion(&h)
  FALSE

VERIFICATION FAILED
